test/integration/targets/ansible-playbook-callbacks/runme.sh

```diff
@@ -5,6 +5,7 @@ set -eux
 export ANSIBLE_CALLBACK_PLUGINS=../support-callback_plugins/callback_plugins
 export ANSIBLE_ROLES_PATH=../
 export ANSIBLE_STDOUT_CALLBACK=callback_debug
+export ANSIBLE_HOST_PATTERN_MISMATCH=warning
```

Comment:
I still think this should be set for the `all-callbacks.yml` playbook only as opposed to the whole script in case more tests are added in the future but not a blocker.